Finish/MonkeyFinder/Model/Monkey.cs

```diff
@@ -9,4 +11,10 @@ public class Monkey
     public int Population { get; set; }
     public double Latitude { get; set; }
     public double Longitude { get; set; }
+}
+
+[JsonSerializable(typeof(List<Monkey>))]
+internal partial class MonkeyContext : JsonSerializerContext
```

Comment:
Consider making this sealed 🙂
```suggestion
internal sealed partial class MonkeyContext : JsonSerializerContext
```